src/test/resources/Logout.feature

```diff
@@ -0,0 +1,22 @@
+Feature: User can logout
+  As a user
+  I want to logout
+  So I can see login screen
+
+  Scenario: User can logout after login
```

Comment:
chỉ cần viết thế này là đủ rồi:` Scenario: User can logout the app`

Reply:
cũng done nhé